content/django-vs-flask.md

```diff
@@ -0,0 +1,71 @@
+# Django vs Flask: which one should you choose for your project?
+Django and Flask are among the most popular Python web frameworks.
+In this article we're going to review their main strengths and weaknesses to help you decide which one is the better fit for your project.
+
+
+# Django
+![django logo](images/flask-vs-django-django-logo.png)
+## Approach
+Django opts for an all batteries included approach.
+From a powerful ORM to an authentication system or a templating engine, Django takes care of it all without the need to learn any extra libraries.
+
+It also has the concept of pluggable apps, that means that you can drop an app folder inside your project and get its functionality right away
+with very little configuration.
+
+## Main features
+- Django has a very powerful ORM that exposes a lot of SQL to the python programming language.
+- Authentication and Authorization built in, without any extra configuration.
+- A built in admin site.
+- A built in mailing system.
+
+## Pitfalls
+- Tightly coupled with its toolset, Django is sort of difficult to use in combination with other tools (replacing the form library for example).
+- The default scaffolding, while not bad for small projects, does not scale without modification when trying to manage things such as multiple environments.
+- It implements the Model View Template pattern, and requires some effort to implement other architecture types with it.
+
+
+# Flask
+![flask logo](images/django-vs-flask-flask-logo.png)
+## Approach
```

Comment:
What about merging this with the Django approach into one section of the post ? That way you can avoid repeating subtitles and the reader can compare right there. This would apply for `Main features` and `Pitfalls` as well.

Reply:
i actually wrote it that way to let the user read and study one framework at a time and make a decision about what to use after reading both.
I think if you're evaluating both features this is more confortable to read.
Maybe a table summing up the points at the end could work for what you want and as a TLDR as well?